src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,67 @@
+package edu.kis.vh.nursery;
+
+/**
+ * @author 240827
+ *	Default rhymer that works similarly to a stack.
+ */
+public class DefaultCountingOutRhymer {
+
+	public static final int NUMBER = 12;
```

Comment:
Wrong name.
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you used review hints ('Github - PR checkpoint' at wikamp).